index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Amkam</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+    />
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <!-- the toolbar (or header): Section 1 -->
+    <header>
+      <h1>DevAmkam</h1>
+      <svg
+        class="w-6 h-6 menu-icon"
+        fill="none"
+        stroke="currentColor"
+        viewBox="0 0 24 24"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          stroke-linecap="round"
+          stroke-linejoin="round"
+          stroke-width="2"
+          d="M4 6h16M4 12h16M4 18h16"
+        ></path>
+      </svg>
+    </header>
+
+    <!-- the headline section (right after the header): Section 2 -->
+    <section class="headline">
+      <div>
+        <h1 class="intro-text">I’m Amadu Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <div class="connect-link"><a href="">Let's Connect</a></div>
+        <div>
+          <a href="https://twitter.com/DevAmkam" target="blank"
+            ><i class="fab fa-twitter network-link"></i
+          ></a>
+          <a href="https://www.facebook.com/amadus.kamara.7" target="blank"
+            ><i class="fab fa-facebook-square network-link"></i
+          ></a>
+          <a
+            href="https://www.linkedin.com/in/amadu-kamara-3b60a25b"
+            target="blank"
+            ><i class="fab fa-linkedin-in network-link"></i
+          ></a>
+          <a href="https://github.com/AmaduKamara" target="blank"
+            ><i class="fab fa-github-square network-link"></i
+          ></a>
+        </div>
```

Comment:
- [ ] **[REQUIRED CHANGES]** Could you please change the social media icons list to be a `<ul>`? The reason for this is to ensure that we are using the HTML semantics - every time that you have a list of similar or identical elements, it is a good practice to use either an ordered list or an unordered list.